pkgs/development/tools/continuous-integration/drone/default.nix

```
@@ -1,22 +1,17 @@
-{ stdenv, fetchFromGitHub, buildGoPackage, go-bindata, go-bindata-assetfs }:
+{ stdenv, fetchFromGitHub, buildGoModule }:
 
-buildGoPackage rec {
-  pname = "drone.io";
-  version = "0.8.6-20180727-${stdenv.lib.strings.substring 0 7 revision}";
-  revision = "c48150767c2700d35dcc29b110a81c8b5969175e";
+buildGoModule rec {
+  name = "drone.io-${version}";
+  version = "1.3.1";
   goPackagePath = "github.com/drone/drone";
 
-  # These dependencies pulled (in `drone` buildprocess) via Makefile,
-  # so I extracted them here, all revisions pinned by same date, as ${version}
-  goDeps= ./deps.nix;
-
-  nativeBuildInputs = [ go-bindata go-bindata-assetfs ];
+  modSha256 = "09mg1lrq93jpl98yp51mpqziarl5dgwm3ifi7f16lchvyy192k3f";
 
   src = fetchFromGitHub {
     owner = "drone";
     repo = "drone";
-    rev = revision;
-    sha256 = "0miq2012nivvr1ysi3aa2xrr5ak3mf0l3drybyc83iycy0kp4bda";
+    rev = "e1ed2623b15e1ab77a468f07cf28307423c0fc88";
```

Use `v1.3.1` tag instead:
```suggestion
    rev = "v${version}";
```

---

I agree about using tagged revision, if it possible here.

Also if one of you want to overtake maintainership of drone, you are welcomed. I not use it actively at the moment.

---

Thanks for your feedback, I've switched to a tagged revision and updated to the maintainers list.